migrating_from_ocp_3_to_4/planning-migration-3-4.adoc

```diff
@@ -148,6 +148,13 @@ If your {product-title} 3.11 cluster used the `ovs-subnet` or `ovs-multitenant`
 
 For more information, see xref:../networking/network_policy/about-network-policy.adoc#about-network-policy[About network policy].
 
+[discrete]
+==== OVN-Kubernetes as the default networking plug-in
+
+The default networking plug-in for {product-title} 3.11 was OpenShift SDN. In {product-title} {product-version}, OVN-Kubernetes is now the default networking plug-in.
```

Comment:
In {product-title} 3.11, OpenShift SDN was the the default networking plug-in in Red Hat OpenShift Networking.